rpc/json/service_test.go

```diff
@@ -70,7 +71,7 @@ func TestREST(t *testing.T) {
 		{"invalid/missing param", "/block", http.StatusOK, int(json2.E_INVALID_REQ), `missing param 'height'`},
 		{"valid/no params", "/abci_info", http.StatusOK, -1, `"last_block_height":345`},
 		// to keep test simple, allow returning application error in following case
-		{"valid/int param", "/block?height=321", http.StatusOK, int(json2.E_INTERNAL), `"key not found"`},
+		{"valid/int param", "/block?height=321", http.StatusOK, int(json2.E_INTERNAL), "error loading hash by height 321: key not found"},
```

Comment:
I think that it might not be "stable" to include block number here.
```suggestion
		{"valid/int param", "/block?height=321", http.StatusOK, int(json2.E_INTERNAL), "error loading block hash for height"},
```